### ISSUE TITLE
Are removal of parent classes/interfaces to be considered BC breaks?

### ISSUE THREAD

**Issue by Nyholm** [Roave/BackwardCompatibilityCheck#106]
```php
class Request implements PSR7RequestInterface
{
  // ...
}
```

If my `Request` class stops implementing `PSR7RequestInterface`, isnt that a BC break that should be detected?

```php
class Request
{
  // ...
}
```

**Comment by Ocramius** [comment]
Yes, that's a BC break: so is removing any ancestor, as type checking will fail in consumers.

**Comment by Ocramius** [comment]
And yes, it is not implemented, but can be done trivially - do you want to try it, or should I give it a shot?

**Comment by Nyholm** [comment]
Im happy if you do it and I'll send the next hour implementing this excellent tool on more open source libraries.

**Comment by Nyholm** [comment]
Okey, great. 

My CI implementation do not care of new major versions of this library. I think it is a good idea to blindly trust that this library does not make anything weird. So until Im proven wrong Im just doing this: 

https://github.com/php-translation/extractor/pull/123/files

**Comment by Ocramius** [comment]
Handled in #109
